tests/test_external_search.py

```diff
@@ -189,7 +189,7 @@ def setup(self):
 
             self.search.bulk_update(works)
 
-            time.sleep(1)
+            time.sleep(2)
```

Comment:
I want to see if this fixes https://github.com/NYPL-Simplified/server_core/issues/353